src/flamegpu/gpu/CUDAAgentModel.cu

```diff
@@ -81,104 +83,136 @@ bool CUDAAgentModel::step() {
         /*! for each func function - Loop through to do all mapping of agent and message variables */
         for (const std::shared_ptr<AgentFunctionData> &func_des : functions) {
             auto func_agent = func_des->parent.lock()->description;
-            if (!func_agent) {
```

Comment:
@mondus Your visual studio settings seem to be murdering indentation (replacing spaces with tabs, won't pass linting)

Reply:
Yeah I think it defaults to tab.

Reply:
Done there will be a few more linting errors to still fix